runtime/lua/vim/lsp/buf.lua

```diff
@@ -30,13 +33,17 @@ end
 --- In the floating window, all commands and mappings are available as usual,
 --- except that "q" dismisses the window.
 --- You can scroll the contents the same as you would any other buffer.
+---
+--- Note: when using `apply_highlight`, ensure that the |hl-LspHoverRange| highlight is defined or
```

Comment:
What if instead of requiring this we just link it to `Visual` by default?

Reply:
Now at least it is linked to `LspReferenceText` which links to `Visual`; would you prefer I link it directly to `Visual`?

Reply:
Oops my bad I missed that. It's fine to link it to `LspReferenceText`, but I would then change this note to "If using `apply_highlight`, the hovered range will be highlighted with |hl-LspHoverRange|". That's enough to let users know of how they can customize it.